MaxText/layers/quantizations.py

```diff
@@ -263,13 +264,12 @@ def __call__(self, eqn, *args, **kwargs):
     k = jnp.asarray(k, comp_dtype)
     x = jnp.asarray(x, comp_dtype)
 
-    x_qdq = fp8_ops.in_qdq(comp_dtype, self.e4m3_dtype, x, self.input_scale.value, self.input_amax_history.value)
-    k_qdq = fp8_ops.in_qdq(comp_dtype, self.e4m3_dtype, k, self.kernel_scale.value, self.kernel_amax_history.value)
+    x_qdq = fp8_ops.in_qdq(self.e4m3_dtype, x, self.input_scale.value, self.input_amax_history.value)
```

Comment:
Should we still keep the comp_dtype here? same as bellow.

Reply:
Ohhhh https://github.com/google/flax/blob/main/flax/linen/fp8_ops.py#L243 has two types whereas mine only has one type. Let me check the dependencies to ensure they require this higher version then I'll revert my change here.